app/helpers/audit_helper.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-module AuditHelper
-  def activity(type_of_activity, args = nil)
+module AuditHelper # rubocop:disable Metrics/ModuleLength
+  def activity(type_of_activity, args = nil) # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
I guess it's good having it in the code so we know that there's a problem, but would also be good to think about how we might fix it 😂

Reply:
Though this is def a step towards that!

Reply:
Sorry, one final thing, do you need to enable it again or is it just on a file by file basis?

Reply:
Yeah, my thinking in these changes is that `.rubocop_todo.yml` is a bit too out-of-sight, out-of-mind and if we disable things in there we may never think about it again. Especially since that disables it for the whole file, not just individual lines/methods that are an issue.

The `disable` comments apply for the current line if they're at the end of the line, or they apply to a range of lines if they're on their own (and then they need an `enable` comment later. I usually prefer the single-line form, but in a couple of places here I've used the multiline form, because the single-line form got flagged as unnecessary, then when it was removed it showed the error I was trying to disable again — thanks rubocop!